src/app/components/CustomTextInput/index.js

```diff
@@ -73,7 +73,7 @@ CustomTextInput.defaultProps = {
   keyboardType: 'default',
   multiline: false,
   bottomBorder: false,
-  maxHeight: 200
+  maxHeight: 200,
```

Comment:
add `refRatioScale(200)` in `src/constants/dimensions.tsx` and import here

Reply:
This file is modified because `the prettier` had a wrong configuration.

Reply:
this moment ignore this file